src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1202,6 +1203,13 @@ private function registerSerializerConfiguration(array $config, ContainerBuilder
             $definition->addTag('serializer.normalizer', array('priority' => -900));
         }
 
+        if (class_exists('Symfony\Component\Serializer\Normalizer\ConstraintViolationListNormalizer')) {
```

Comment:
Had the same issue with https://github.com/symfony/symfony/pull/20567/files#diff-1cf8b3cc93de9c5674101413d27a9bf0R47, at first i thought; lets be consistent. On 2nd thought i chose to _just_ use `::class` notation for new code, as it's the better syntax en exposes class deps (ie. it improves usage tracking, and therefor maintenance).